packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -123,13 +148,12 @@ export class ZoweJobNode extends ZoweTreeNode implements IZoweJobTreeNode {
                     label: vscode.l10n.t("There are no JES spool messages to display"),
                     collapsibleState: vscode.TreeItemCollapsibleState.None,
                     parentNode: this,
+                    profile: this.getProfile(),
                 });
-                noSpoolNode.iconPath = null;
+                noSpoolNode.iconPath = undefined;
```

Comment:
Technically any falsy value will work, but I felt it was appropriate to change it since there were other references to `iconPath = undefined` from a few years ago, and because I've noticed the VS Code team [falling back to `undefined`](https://github.com/microsoft/vscode/blob/8558da133548b1a0b83f038e103ffa7f84f47da4/src/vs/workbench/api/common/extHostTreeViews.ts#L840) as well 😋 

and no problem!